src/Diagnostics/FxCop/Desktop.Analyzers/CSharp/MSInternal/DoNotCatchCorruptedStateExceptionsAnalyzer.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Collections.Immutable;
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+using SDL.Analyzer.Common;
+
+namespace Desktop.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public sealed class CSharpDoNotCatchCorruptedStateExceptionsAnalyzer : DoNotCatchCorruptedStateExceptionsAnalyzer
+    {
+        private static readonly ImmutableArray<SyntaxKind> nodeKindsOfInterest =
```

Comment:
This should be named `s_nodeKindsOfInterest` as per the [C# Coding Style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-style.md) guidelines.